# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PAPZCTreeManager.ipdl
# Commit: febdcf7986a3
# Full Hash: febdcf7986a37651188476cd8253d20e9cea443c
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 00d1c650d88f -r febdcf7986a3 gfx/layers/ipc/PAPZCTreeManager.ipdl
--- a/gfx/layers/ipc/PAPZCTreeManager.ipdl	Wed Jun 22 19:26:58 2022 +0000
+++ b/gfx/layers/ipc/PAPZCTreeManager.ipdl	Wed Jun 22 19:40:17 2022 +0000
@@ -83,6 +83,8 @@
                            Modifiers aModifiers);
 
   async CancelAutoscroll(ViewID aScrollId);
+
+  async NotifyScaleGestureComplete(ViewID aScrollId, float aScale);
 };
 
 } // namespace gfx